### PR TITLE
Add link to portfolio workshop in github-pages workshop

### PR DESCRIPTION
**Pull request by paked**
Just thought this would be a nice addition. You know, get some cross sell going!

### CODE REVIEW

File: workshops/contrib/github-pages/README.md [hunk 1]
```diff
@@ -47,6 +47,9 @@ terminal. (Instructions directly below)
 You should start creating the necessary files for a website. (`index.html`,
 `style.css`, etc)
 
+*If you don't know how to do this, you should check out the
+[portfolio workshop](../../portfolio) to learn how to build your website!*
```
Comment (zachlatta) [inline workshops/contrib/github-pages/README.md]: Can you have this link to the portfolio workshop's README directly?

`../../portfolio/README.md`

**Comment by zachlatta** [comment]
Should we merge it @sethtrei? I think you should have the final call on this.

I'm personally for merging it.

File: workshops/contrib/github-pages/README.md (hunk 1 above)
Comment (paked) [inline workshops/contrib/github-pages/README.md]: @zachlatta Not quite sure I see the benefit.

**Comment by sethtrei** [comment]
All good.

**Pull request closed without merging**